pom.xml

```diff
@@ -1,44 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-
-	<groupId>np.com.lostfound</groupId>
-	<artifactId>lostfound</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<name>lostfound</name>
-
-	<properties>
-		<java.version>11</java.version>
-		<spring.version>3.0.4</spring.version>
-		<maven.compiler.source>1.8</maven.compiler.source>
-     	<maven.compiler.target>1.8</maven.compiler.target>
-		<lombok.version>1.18.26</lombok.version>
-	</properties>
-
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
-			<version>${spring.version}</version>
-		</dependency>
-
-		<dependency>
-			<groupId>org.projectlombok</groupId>
-			<artifactId>lombok</artifactId>
-			<version>${lombok.version}</version>
-			<scope>provided</scope>
-		</dependency>
-	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-				<version>${spring.version}</version>
-			</plugin>
-		</plugins>
-	</build>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>np.com.lostfound</groupId>
+    <artifactId>lostfound</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>lostfound</name>
+
+    <properties>
+        <java.version>11</java.version>
+        <spring.version>3.0.4</spring.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <lombok.version>1.18.26</lombok.version>
+        <spring-boot-starter-data-jpa.version>3.0.0</spring-boot-starter-data-jpa.version>
+        <h2database.version>2.1.214</h2database.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <version>${spring.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>${spring-boot-starter-data-jpa.version}</version>
```

Comment:
use spring version, all the dependencies, under group id "org.springframework.boot" should have the same version.